report_qweb_pdf_watermark/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=============
+Pdf watermark
+=============
+
+This module was written to add watermarks (backgrounds) to PDF reports. Because of the way wkhtmltopdf handles headers and footers in the current versions, it is quite impossible to have a background for the complete page using HTML and CSS. That is why this module inserts the image at the PDF level.
+
+Installation
+============
+
+As PyPDF is not supported in python3, you need to install PyPDF2::
+
+$ pip install pypdf2
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. go to your report
+#. select a PDF or image to use as watermark. Note that resolutions and size must match, otherwise you'll have funny results
+#. You can also fill in an expression that returns the data (base64 encoded) to be used as watermark
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/143/8.0
```

Comment:
11.0

Reply:
seems even with 11.0 I can't test the module.
What should I change the 143 to, to get the right runbot?

Reply:
answering my own question, as long as this is not merged, it will not show up on that runbot